### PR TITLE
First commit of Retrofit2 file & Libraries

### PR DESCRIPTION
**Pull request by mine0697782**
It is a first commit of Retrofit2 class file and libraries.
In the Android Studio, I used retrofit2 to connect to server(sending POST, GET),
but some problems occured continuously so I added Libraries to resolve.
I'm sorry but I couldn't.

### CODE REVIEW

File: AndroidStudioProject/ExeTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public interface GitHubService {
+        @POST("api/login/{Email}")
+        Call<JSONArray> getUserName();
+                //@Path("Email") String userEmail);
+    }
+
+    public static void main(String[] args){
+        Retrofit retrofit = new Retrofit.Builder()
+                .baseUrl("http://52.8.187.167/")
+                //.addConverterFactory()
```
Comment (ethkim) [inline AndroidStudioProject/ExeTest.java]: Remove all commented out code.

File: AndroidStudioProject/ExeTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import retrofit2.Retrofit;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.Path;
+
+public class ExeTest {
+    public interface GitHubService {
+        @POST("api/login/{Email}")
+        Call<JSONArray> getUserName();
+                //@Path("Email") String userEmail);
```
Comment (ethkim) [inline AndroidStudioProject/ExeTest.java]: Remove all commented out code.

File: AndroidStudioProject/ExeTest.java [hunk 3]
```diff
@@ -0,0 +1,42 @@
+package com.example.owner.myapplication;
```
Comment (ethkim) [inline AndroidStudioProject/ExeTest.java]: This is not an android code - why are we checking in this test file?

Comment (mine0697782) [reply AndroidStudioProject/ExeTest.java]: I'll remove all these files and work in Woongjin's branch.

**Pull request closed without merging**